src/sections/header.html

```diff
@@ -1,4 +1,34 @@
-<div>
-    <h1 class="text-blue-800">Header</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
+<!-- desktop -->
+<div class="flex mx-auto justify-between pt-2 pb-2 px-4 fixed z-50 w-full top-0 text-xl mt-6 md:mt-0">
+  <h1 class="font-medium w-28">DE<br />EN</h1>
+  <h2 class="font-bold text-4xl flex items-center justify-center relative px-5">
+    <img class="absolute -z-10 w-full md:hidden" 
+    src="/src/assets/whiteHeader.svg" alt="">
+    Baumeister
```

Comment:
[The use of `alt` attribute on the image is mandatory](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) except if the image is completely irrelevant to the content.
These images are pretty irrelevant but let's just practice it for the sake of practicing it.
 
```suggestion
    <img class="absolute -z-10 w-full md:hidden" 
    src="/src/assets/whiteHeader.svg" alt="no empty alt attributes please">
    Baumeister
```